kong/cmd/utils/nginx_signals.lua

```diff
@@ -12,7 +12,7 @@ local nginx_search_paths = {
   ""
 }
 local nginx_version_pattern = "^nginx.-openresty.-([%d%.]+)"
-local nginx_compatible = version.set(meta._DEPENDENCIES.nginx)
+local nginx_compatible = version.set(unpack(meta._DEPENDENCIES.nginx))
```

Comment:
btw, like this unpack solution. I tinkered about how to do it neater, but didn't think of this. Nice! :+1: